### ISSUE TITLE
Thrown errors causing entire Svelte app to freeze + valid slider step + slider value throws an error

### ISSUE THREAD

**Issue by nyanpasu** [hperrin/svelte-material-ui#255]
**Describe the bug**
He, this is actually a bug [upstream](https://github.com/material-components/material-components-web/issues/7148), but I'm letting you guys here know too as I encountered this bug while using this lib and perhaps there's a better way for us to handle it here as it seems a thrown error just stops svelte entirely in its tracks, causing the entire svelte app to stop working.

**To Reproduce**

Please refer to https://github.com/material-components/material-components-web/issues/7148

Basically:

> Hi, I'm getting this error even for valid values, such as:
> 
> ```
>   min={0.01}
>   step={0.01}
>   value={2500}
> 
> ```
> This is due to a floating point error... and checking if it is divisible by 1...
> 
> ```
> (2500 - 0.01) / 0.01
> > 249998.99999999997

```

**Expected behavior**
Wish the entire app didn't crash due to a single thrown error

**Comment by ThomasOrlita** [comment]
Fixed in v7
